lib/board.rb

```diff
@@ -0,0 +1,55 @@
+class Board
+
+	attr_accessor :cells
+
+	def initialize
+		reset!
```

Comment:
I've seen it before. What setup are you using? And what text editor?
On Tue, May 3, 2016 at 2:00 PM Amy <notifications@github.com> wrote:

> In lib/board.rb
> <https://github.com/learn-co-students/ttt-with-ai-project-v-000/pull/134#discussion_r61926517>
> :
>
> > @@ -0,0 +1,55 @@
> > +class Board
> > +
> > +	attr_accessor :cells
> > +
> > +	def initialize
> > +		reset!
>
> For some reason, the indentations/spacing change when I push to Github. My
> indentations should be two spaces. Wondering if this is a common thing...
> or just me?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/learn-co-students/ttt-with-ai-project-v-000/pull/134/files/c1329a9310498664ad6eac2bce2945a1a3249547#r61926517>
>